junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -573,9 +573,11 @@ public static boolean isPackage(String packageName) {
 		Preconditions.notNull(clazz, "Class must not be null");
 		Preconditions.notNull(sortOrder, "MethodSortOrder must not be null");
 
-		// TODO [#333] Determine if we need to support bridged methods.
-
-		List<Method> localMethods = Arrays.asList(clazz.getDeclaredMethods());
+		// @formatter:off
+		List<Method> localMethods = Arrays.stream(clazz.getDeclaredMethods())
+				.filter(Method::isBridge) // [#333] don't collect bridge methods
```

Comment:
@sormuras I believe you can still use streams here; you just can't use the method reference `Method::isBridge` as-is anymore, and instead you have to use a lamda expression like `method -> !method.isBridge()`.

Or, even better, create a `not(Function)` method (similar to [OpenGamma Strata's Guavate's own `not` method](https://github.com/OpenGamma/Strata/blob/master/modules/collect/src/main/java/com/opengamma/strata/collect/Guavate.java#L182)), put it in `org.junit.platform.commons.util.FunctionUtils.java`, and pass the `Method::isBridge` method ref to it like so:
```java
import static org.junit.platform.commons.util.FunctionUtils.not;
 . . .
List<Method> localMethods = Arrays.stream(clazz.getDeclaredMethods())
    .filter(not(Method::isBridge))
    .collect(toList());
```